### DISCUSSION TITLE
Should NodeSchool continue to exist?

### DISCUSSION THREAD

**Discussion by finnp** [nodeschool/organizers#597] [Polls]
Hey @nodeschool/chapter-organizers,

Just looking through the NodeSchool org, it doesn't seem to me like any chapters are still active. Should we maybe sunset NodeSchool as a whole?

I noticed that:
- Most Workshoppers are out-of-date or unmaintained
- The websites list of chapters is broken
- I don't really see activities in the chapter repositories
- There are very few discussions/questions popping off and the ones that occur are unanswered

It feels to me it would be better to sunset NodeSchool:
- Less confusion for people who might end up on the website to learn node or javascript
- No possibility of the NodeSchool org being used for something harmful by one of the I think 300 admin accounts

Wondering what people think 🤷‍♂️

**Comment by fforres** [comment]
We are actively doing nodeschool events on Santiago, Chile. 
IIRC, SF had a few events recently :)

**Reply by finnp** [reply]
That's great :) Do you use the current workshoppers? Or do you use different resources for teaching?

**Comment by robbiethegeek** [comment]
We have rolled Nodeschool into DC JS Playdays here in the Washington DC
area. That was before the beginning of the pandemic and haven’t run a DC JS
play day during the pandemic.

Robbie Holmes
RobbieTheGeek

**Comment by jaybobo** [comment]
Though I look back on it with lots of fondness...

### It's time to end it.

- The number of resources for learning JS and node has increased exponentially on the web as the ecosystem has evolved and become more mainstream.
- Nodeschool Columbus was rolled into PairColumbus.org in Central Ohio but we haven't had an event or used the curriculum since the pandemic started. I'd imagine many other chapters are in a similar state.

But, I'd recommend archiving allthethings for posterity and pointing future visitors to FCC or MDN.

**Reply by finnp** [reply]
At this point it feels like a lot of people would like to continue with NodeSchool. But I'm wondering, what do you think are the best resources for learning JavaScript/Node right now?

**Comment by RichardLitt** [comment]
> Less confusion for people who might end up on the website to learn node or javascript
> No possibility of the NodeSchool org being used for something harmful by one of the I think 300 admin accounts

Both of these things can be accomplished without asking about the state of Nodeschool in a way that invalidates current efforts (cool that it is still going on in Chile!) and honors past members, I think.

How about we add a banner to the site saying that there may be other resources that are more useful, but these are still available for those who would like them? We can point to current efforts.

As for the admins, let's just make a "Alumni" group on the org with no org rights, and then assign everyone to that unless they're actively organizing a chapter. That helps harden the org and allows for people to fade away on their own.

**Reply by finnp** [reply]
What do you think is the best process of determining if a chapter is active?

**Reply by RichardLitt** [reply]
Probably email all of the admins, and note unresponsive if no response in a month or so. :/ I don't see how to do this easily.

**Comment by gartox** [comment]
At Villahermosa, Tabasco, We had a couple of events over the weekend, for which have been a great way to teach about Js, Node, and development in general. But yes, its a good way to do something else, to evolve.

**Reply by finnp** [reply]
Do you use nodeschool workshoppers? If not, what resources do you use?

**Comment by bnb** [comment]
I'd love to continue running them in NYC, but personally found it super challenging to support learners with learnyounode and some of the other workshoppers - it didn't feel like my time was most effectively spent helping attendees with that specific material, and it didn't feel like I was effectively providing the best experience for those who were attending.

I'd be totally willing to spend personal time and energy on a total rewrite of the different materials if that's something others would be interested in.

**Reply by bnb** [reply]
Also: +1 to alumni team.

**Reply by Qard** [reply]
I'm also happy to help with getting some up-to-date content together. In Vancouver the vast majority of students go for javascripting or learnyounode so more effort on getting the core workshops in good shape would be great.

**Reply by finnp** [reply]
Would love to see people update the learnyounode materials. I think @martinheidegger was one of the last people actively updating it. One challenge with it might be all the translations, so potentially starting from scratch could be worthwhile.

**Comment by tdd** [comment]
Hey all,

The Paris chapter had a glorious, 3-year run of regular full-day, free-catering events at the gorgeous Mozilla Paris office.

But their moving to a less practical place and then the pandemic basically stopped it.

Even the Node.js Paris meetup is in a semi coma these days.

I agree the whole thing is now out of touch. Workshoppers are sorely outdated, and tons of resources exist to learn Node online with zero install.

Alumni and a banner seem like good steps!

On Mon, Apr 3, 2023, at 07:12, Tierney Cyren wrote:
> 
> 
> Also: +1 to alumni team.
> 
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/orgs/nodeschool/discussions/597#discussioncomment-5505676>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAA5KB4THIWYTI5U4CJP63W7JL5NANCNFSM6AAAAAAWQP4G5Y>.
> You are receiving this because you are on a team that was mentioned.Message ID: ***@***.***>
> 

--
Christophe Porteneuve
***@***.***

**Comment by SomeoneWeird** [comment]
Alumni team seems like a good idea. Lets leave this issue open for another week to gather comments and go from there?

I do think continuing to keep the 'entity' around is a good move. Devoting resourcing to update existing workshops (or remove them as outdated(?)) is likely better than spinning down imo

**Comment by amitsin6h** [comment]
Hello all,

I was the organiser for Nodeschool/Jharkhand but due to the pandemic, I was
not able to give much time to the community and also sudden recession.

**Comment by zhdzmn** [comment]
Hello all,
The Dhaka chapter has been silent for some time now since we had a lot of
different focus in the community. we would actually want to restart this as
soon as possible. The pandemic did not help but I do believe this banner
means alot.
I am ok with the alumni concept sounds good to me.
Zahid

On Mon, Apr 3, 2023 at 11:34 AM Amit singh ***@***.***> wrote:

> Hello all,
>
> I was the organiser for Nodeschool/Jharkhand but due to the pandemic, I was
> not able to give much time to the community and also sudden recession.
>
> On Mon, Apr 3, 2023 at 10:57 AM Adam Brady ***@***.***> wrote:
>
> > Alumni team seems like a good idea. Lets leave this issue open for
> another
> > week to gather comments and go from there?
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/orgs/nodeschool/discussions/597#discussioncomment-5505751
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ADCO42UPLCIKXDT4KCNW2OTW7JNS3ANCNFSM6AAAAAAWQP4G5Y
> >
> > .
> > You are receiving this because you are on a team that was
> mentioned.Message
> > ID: ***@***.***
> > com>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/nodeschool/discussions/597#discussioncomment-5505793>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAX7Y5GHS547MG4U4GK4H5LW7JOPVANCNFSM6AAAAAAWQP4G5Y>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: ***@***.***
> com>
>


-- 
--
A. K. M. Zahiduzzaman

**Comment by Xabadu** [comment]
Hi everyone, Thanks for setting up this discussion.

I'm the organizer of the San Francisco chapter. We ran monthly events until the beginning of the pandemic, when we moved to virtual events and activities through Discord. We resumed in-person gatherings in 2022, and took a brief hiatus closer to the end of the year due to some issues with our venues, but we are planning to get back soon™ since there's still interest from our community in these type of events.

However, one thing I wanted to mention is that our group, and particularly events, have been mutating quite a bit since I'd say 2018 maybe? Few different reasons, but mostly the fact that a significant part of our attendees are people either trying to learn coding/web development in general, or folks looking to transition into tech from other careers, and for them I think the materials from NodeSchool present a lot of friction, particularly with those unfamiliar using a terminal, installing packages, etc, so the amount of people actually using them during the events is really low/none. We've also noticed that it tends to drive people away, so as of last year, we've been intentionally recommending other resources instead, while at the same time trying to kickstart a couple of efforts on our end to contribute and update some of these resources, but sadly life gets in the way and volunteer time is rare, so we've been unable to do so, but more than happy to share our ideas and concepts if that's helpful.

That being said though, I think NodeSchool as a concept/name/brand, whatever you wanna call it, still holds a lot of value in terms of discoverability for folks trying to find community events that are more hands-on coding focused and not talks focused. In our case at least, it's a big part of how people end up joining us, but also very aware that it might not be the case for everyone else.

Big +1 to mark these resources and point to others to set the right expectations btw and creating the Alumni group, that'd be super helpful.

**Reply by finnp** [reply]
In Berlin we already stopped doing events in I think 2018, but I had a similar experience. A lot of people came to NodeSchool to learn JavaScript of web development in general. So often I would point people to different resources to learn basics or encourage them to create small projects with html/css/js.

What kind of materials do you usually use? I'm wondering if we could add some of them to the NodeSchool homepage. 

Also feel free to go ahead and add a link to the NodeSchool SF chapter to the nodeschoo.io website if you're restarting. For now I just removed links to all chapters. I think given that the number of active chapters seems to be low it could even be quite a prominent link. Could even be a fun project for learners ☺️

**Comment by Qard** [comment]
Vancouver chapter is active. We were inactive for awhile during the pandemic but have been active again for several months now. We're typically getting about 40 students per event. We _have_ found many electives to be out-of-date, but the core workshops are still largely relevant and mostly all working. 90%+ of our students go straight for javascripting or learnyounode anyway, so more focus on the core stuff would be all that's needed.

I'd be 👎🏻 to terminating NodeSchool. IMO it just needs new people in charge to get things moving again. Might be a good idea to hand it over to OpenJS Foundation.

**Reply by sclarson** [reply]
> the core workshops are still largely relevant and mostly all working. 90%+ of our students go straight for javascripting or learnyounode anyway, so more focus on the core stuff would be all that's needed.

I definitely haven't had an event since the pandemic but had also tapered off some before that. That said I cannot echo strong enough how great it is to have learnyounode and similar core modules to point people at.

**Reply by Qard** [reply]
The Vancouver chapter was quite active right up to the pandemic and is coming back quite strong. There's _huge_ demand here at the moment for the sort of study group that NodeSchool provides.

**Comment by RichardLitt** [comment]
On a small, possibly related note to OpenJS: Open Source Collective would be happy to fiscally host any Nodeschool chapter of more than a few score members. Just let me know.

**Reply by fforres** [reply]
That'd be wonderful. Do you folks do that through something like opencollective?

**Reply by RichardLitt** [reply]
Yes. https://opencollective.com/opensource is a 501c6 that uses Open Collective to fiscally host projects. We're happy to host groups that are based on open source and are active and need someone to just hold their finances and allow people to donate to them and to invoice for reimbursements/work done. https://docs.oscollective.org/what-we-offer

Disclaimer: I work at OSC.

**Comment by klequis** [comment]
I also think the brand has value in drawing people in. However, as I think one of @Xabadu 's points is saying, I'm not convinced that NodeJS is a wide enough subject for in-person, hands-on events to fulfill what people are looking for in a group that plans to meet regularly, say a monthly format as many of us aim for.

I enjoy writing tutorials and exercises so if we get to a vision of what they should be in the future keep me looped in.

**Reply by klequis** [reply]
should have said "is NOT a wide enough subject for ..."

**Comment by greelgorke** [comment]
Hi!

I'm the founder of the Hamburg chapter. Back in the day, we had an interest
and ran a few events. I dropped out from the scene a few years ago but kept
an outside perspective.

The Hamburg chapter always has been embedded into the wider community here.
In Hamburg, there is an active general JS community. but also different
other communities with different Web-related Topics. Plus, there are some
players who care about tech education like Open Tech School. All of this is
quite interconnected, because, let's face it, the web was always more than
the sum of its parts. For me, Nodeschool has always been an extension of
our Node.js meetup, which was a branch of the JS meetup. One thing I've
noticed, however, was that Nodeschool attracted people from the outside,
not so much of the established people.

So, I think it's only natural when a good running Nodeschool chapter
evolves into a more general format like meetup.

And here is my point, I think. Every chapter should strive for one of the
two roads. Either evolving beyond into a more general community if there is
none yet. Or partner up and collaborate with existing communities,
devolving Nodeschool into an offering by those communities.

Nodeschool and the workshoppers were very valuable to start and get
traction, but I think they were always too limited to sustain a community
just around it. This needs to be integrated into a bigger picture, but on a
local chapter basis.

I also wonder, if the brand really is that valuable like some said here.
I'd be careful, we're prone to survivorship bias here.

Am Di., 4. Apr. 2023 um 02:30 Uhr schrieb klequis ***@***.***

**Comment by greelgorke** [comment]
Additional point: In my experience, the communities in very active areas
tend to segment themselves sometimes. In short terms, this might have the
benefit of deeper focus, but in the long run, it added to attention fatigue
and made it harder for each group to find enough contributors. So for me,
it makes sense to become a part of a bigger more general group for this
reason too.

Am Di., 4. Apr. 2023 um 11:18 Uhr schrieb Gregor Elke ***@***.***

**Comment by SomeoneWeird** [comment]
> No possibility of the NodeSchool org being used for something harmful by one of the I think 300 admin accounts

Just to clarify too, there are 500+ members of the org, but only 15 owners

**Reply by finnp** [reply]
Good point!  Also I don't think that point is that important anyway, since I don't see a big risk here.

**Comment by jaybobo** [comment]
@finnp if it ends in a 50/50 split, we trust you to make the decision for the community. we'll be ok.

**Comment by finnp** [comment]
I haven't had time to look through all of the responses but I just wanted to make clear that the pill wasn't meant as a majority vote 😅 

I think if there is just a few that still organize nodeschools we should probably keep the website/organization.

I did for now update the website to remove the chapter list that is very outdated. However if people want to go ahead and reads it with active chapters or even add an alumni section I think that would be cool!

**Reply by Qard** [reply]
Probably worth some sort of liveness check automation to automatically move chapters to alumni if the organizers don't take some action to keep them in the list on a semi-regular basis.

**Reply by finnp** [reply]
Yeah. I'm wondering though if it's worth it to have an automated system. Kind of depends of the number of chapters that want to continue hosting NodeSchool events.

**Comment by NorthDecoder** [comment]
@finnp I was hosting nodeschool [Tacoma](https://github.com/nodeschool/tacoma) for a while, which also has an outdated nodeschool [webpage](https://nodeschool.io/tacoma/). I cancelled the TacomaJS meetup group years ago and have no intention of starting it again.

I do not quickly see how to remove the Tacoma nodeschool repository, but if the owners of the organization want to do it for me, this is the official authorization from me for you to do so.  If you need me to do some additional action, please provide the details and I will comply 😎

**Comment by riyadhalnur** [comment]
+1 to alumni. I really haven't had time to admin the org anymore. Rather than spinning down, I would rather keep the website up and point people to well-known/popular resources. i.e use this as a starting point.

**Comment by satanas** [comment]
Hi all,

I was the organizer of the Seattle chapter for several years but due to the
pandemic we stopped. I quit as an organizer on meetup.com and I don't have
intentions of resuming any activities, so the alumni group and the banner
idea sounds good to me.

Cheers!

On Fri, Apr 7, 2023 at 12:47 PM Riyadh Al Nur ***@***.***>
wrote:

> +1 to alumni. I really haven't had time to admin the org anymore. Rather
> than spinning down, I would rather keep the website up and point people to
> well-known/popular resources. i.e use this as a starting point.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/nodeschool/discussions/597#discussioncomment-5556218>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAHH32NJWDODOT5SFVFZ33XABVOZANCNFSM6AAAAAAWQP4G5Y>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: ***@***.***
> com>
>


-- 
Wil A. Alvarez
Linux Counter #415026
Debian Counter #259
http://satanas.io
